packages/evm/test/runCall.spec.ts

```diff
@@ -725,37 +725,67 @@ describe('RunCall tests', () => {
       common,
     })
     for (const opcode of ['f1', 'f2']) {
-      // Code to either CALL or CALLCODE into the own address, with value 1
-      // JUMPDEST added at the start to track how much CALL(CODE)s are made
-      const contractCode = hexToBytes(`0x5B6000808080600130611a90${opcode}`)
-      const contractAddress = Address.fromString('0x000000000000000000000000636F6E7472616374')
-      await evm.stateManager.putContractCode(contractAddress, contractCode)
+      // Code to either CALL or CALLCODE into AACC empty contract, with value 1
 
-      const account = await evm.stateManager.getAccount(contractAddress)
-      account!.balance = BIGINT_100
+      const callCodeAddress = Address.fromString('0x000000000000000000000000000000000000aaaa')
+      const callCode = hexToBytes(`0x6000600060006000600161AACC611a90${opcode}`)
 
-      await evm.stateManager.putAccount(contractAddress, account)
+      const gas = (11600).toString(16).padStart(4, '0') // Enough gas so less than required for CALL/CALLCODE before adding call stipend
```

Comment:
Although we were not vulnerable to this bug in CALL previously, the amount of gas necessary for CALL to trigger this is different than the gas for CALLCODE (which is 11600 for Shanghai)

Reply:
(Likeliky you can do `for [opcode, gas] of [['f1', GasCall], ['f2', 11600]] {`)